src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -135,8 +135,23 @@ def get_fuzz_task_payload(platform=None):
   if not mappings:
     return None, None
 
+  logs.log(f'Mappings: {mappings}')
+  # The environment variable containing a list of comma-separated jobs.
+  # E.g: "libfuzzer_asan_android_host,afl_asan_android_host,..."
+  jobs_selection = environment.get_value('HOST_JOB_SELECTION')
+  if jobs_selection:
+    jobs = get_job_list(jobs_selection)
+    selected_mappings = [entity for entity in mappings if entity.job in jobs]
```

Comment:
Just re-set mappings. and get rid of the else condition.

Reply:
Done